README.md

```diff
@@ -1 +1,21 @@
 # Assignment
+## Evaluate Importance and Priority
+#### Prioritization Factors
+*	**Financial value of requirements;** Major factor to be considered in prioritizing requirements. The value could be expressed as new revenue, incremental revenue, or as operational efficiency.
+*	**Cost of developing requirements;** Essential factor to be considered by the product owner. Value and cost together indicate the Return on Investment (RoI) for the requirements.
+*	Amount and **significance of functionality** that will be gained while working on the requirements.
+*	Understanding **level of risks** involved in introducing the new features.
+
+#### Prioritization Techniques
+Follow the _**relative weighting scheme**_ when trying to prioritise any task. It is a simple model where prioritization is done based on all the factors of the task. Major factors to be considered when weighing prioritization are:
+*	The **value of feature** and the negative impact that might be caused by the absence of the feature
+*	Based on the collective judgment from product owner and the development team a weight is assigned to each feature in the following way (a scoreboard from 1 to 9 is usually used)
```

Comment:
judgement from the product owner and the development team, a weight

comma after development team, the before product owner